Dockerfile

```diff
@@ -1,22 +1,35 @@
-FROM ubuntu:18.04
-MAINTAINER me@mehthehedgehog.be
+FROM ubuntu:18.04 as builder
```

Comment:
I guess we had better use `Ubuntu:19:04`.  It is more consistent with libraries installed on our laptops thus we would be able to use the image in more different ways.  I.e. we would be able to use locally built libraries/binaries inside a container.

Reply:
done